provider/labels.go

```diff
@@ -15,18 +17,24 @@ import (
 // tfbridge.SchemaInfo.
 func fixLabelNames(prov *tfbridge.ProviderInfo) {
 	// Recursively applies the label fix
-	var apply func(string, shim.Schema, *tfbridge.SchemaInfo)
-	apply = func(key string, m shim.Schema, info *tfbridge.SchemaInfo) {
+	var apply func(string, shim.Schema, *tfbridge.SchemaInfo, *[]resource.PropertyPath, resource.PropertyPath)
+
+	apply = func(key string, m shim.Schema, info *tfbridge.SchemaInfo,
+		paths *[]resource.PropertyPath,
+		currentPath resource.PropertyPath,
+	) {
 		if key == "terraform_labels" {
 			info.Name = "pulumiLabels"
+			// Gather all paths to pulumiLabels
+			*paths = append(*paths, currentPath)
```

Comment:
This should reference the pulumi name, not the tf name.
```suggestion
			*paths = append(*paths,
				append(currentPath[:len(currentPath)-1], info.Name))
```